dist/css_tagger.min.js

```diff
@@ -1,3 +1 @@
-// Minified on http://closure-compiler.appspot.com/ using Advanced settings
```

Comment:
Not one over another. Sounds like `uglifyjs` is easier to work with. Do you mind documenting it somewhere?